test/activity-test.js

```diff
@@ -13,7 +14,7 @@ describe('Activity', function() {
   let userRepo;
   let activity;
 
-  beforeEach(function() {
+  beforeEach(function() { // create smaller sample, probably use before
```

Comment:
Definitely use the sample data set instead of this hard coded data.

Reply:
Yep! Just needed that file merged to start using it